app/services/event_service.rb

```diff
@@ -0,0 +1,11 @@
+class EventService
+  def self.find_by_zip(zipcode)
+    # link to docs http://api.eventful.com/docs/events/search
+    # refactor the faraday.get and params
+    # I think the March piece should really be Next+Month, get it working then try it out
+    response = Faraday.get("http://api.eventful.com/rest/events/search?l=#{zipcode}&t=March+2017&app_key=#{ENV['APP_KEY']}")
+    hash = Hash.from_xml(response.body.gsub("\n", ""))
```

Comment:
interesting approach here. if you had replaced "rest" in the above url to "json", you would have received json back from your request 👍  then you could've done the normal parsing of the json!